vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -172,15 +172,15 @@ def parse_endpoint(entity)
       new_result
     end
 
-    def parse_replication_controllers(container_replication_controller)
-      new_result = parse_base_item(container_replication_controller)
+    def parse_replication_controllers(container_replicator)
```

Comment:
> worth to rename the method as well

@abonas But the rest of the ```parse_something``` methods use the k8s names like ```parse_pod``` instead of ```parse_container_group```